projects/kubernetes-sigs/image-builder/build/setup_packer_configs.sh

```diff
@@ -74,13 +74,15 @@ export ETCDADM_HTTP_SOURCE=${ETCDADM_HTTP_SOURCE:-$(build::common::get_latest_ek
 # TODO: fix etcdadm build to set correct version
 export ETCDADM_VERSION='v0.0.0-master+$Format:%h$'
 export CRICTL_URL=${CRICTL_URL:-$(build::common::get_latest_eksa_asset_url $ARTIFACTS_BUCKET 'kubernetes-sigs/cri-tools')}
-export CRICTL_SHA256=$(curl $CRICTL_URL | sha256sum | cut -d ' ' -f1)
+export CRICTL_SHA256="$CRICTL_URL.sha256"
 
 envsubst '$IMAGE_REPO:$KUBERNETES_ASSET_BASE_URL:$KUBERNETES_VERSION:$KUBERNETES_SERIES:$CRICTL_URL:$CRICTL_SHA256:$ETCD_HTTP_SOURCE:$ETCD_VERSION:$ETCDADM_HTTP_SOURCE:$ETCD_SHA256:$ETCDADM_VERSION:$KUBERNETES_FULL_VERSION' \
     < "$MAKE_ROOT/packer/config/kubernetes.json.tmpl" \
     > "$OUTPUT_CONFIGS/kubernetes.json"
 
-export ADDITIONAL_PAUSE_IMAGE=$(echo $PAUSE_IMAGE | cut -f1 -d":"):$ADDITIONAL_PAUSE_IMAGE
+ADDITIONAL_PAUSE_IMAGE_VERSION_BASE_URL=$(build::eksd_releases::get_eksd_kubernetes_asset_base_url $ADDITIONAL_PAUSE_IMAGE_FROM)
+ADDITIONAL_PAUSE_KUBERNETES_VERSION=$(build::eksd_releases::get_eksd_component_version "kubernetes" $ADDITIONAL_PAUSE_IMAGE_FROM)
+export ADDITIONAL_PAUSE_IMAGE=$ADDITIONAL_PAUSE_IMAGE_VERSION_BASE_URL/$ADDITIONAL_PAUSE_KUBERNETES_VERSION/bin/linux/amd64/pause.tar
```

Comment:
I want to make sure if this pause.tar is an OCI image tarball. I have a faint memory of this being one of the issue why we pulled from ECR as opposed to tarball.

Reply:
it is, i tested this.

Its created [here](https://github.com/aws/eks-distro/blob/main/Common.mk#L69) with output_type being oci.